x-pack/plugins/security_solution/server/usage/collector.ts

```diff
@@ -6,12 +6,12 @@
 
 import { LegacyAPICaller, CoreSetup } from '../../../../../src/core/server';
 import { CollectorDependencies } from './types';
-import { DetectionsUsage, fetchDetectionsUsage } from './detections';
+import { DetectionsUsage, fetchDetectionsUsage, defaultDetectionsUsage } from './detections';
 import { EndpointUsage, getEndpointTelemetryFromFleet } from './endpoints';
 
 export type RegisterCollector = (deps: CollectorDependencies) => void;
 export interface UsageData {
-  detections: DetectionsUsage;
+  detections: DetectionsUsage | {};
```

Comment:
I think this will always be `DetectionsUsage`, right?

Reply:
Yea, forgot to remove this when I updated. It's out now. Thanks!